walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -1,68 +1,107 @@
-          <div class="projects-container">
-            <div class="description"></div>
-            <div class="description"></div>
-            <div class="description"></div>
+          <div class="description">
+            <p>devbootcamp work experience</p>
+          </div>
+        </div>
+        <div class="work">
+          <div class="image">
+            <img src="images/utsa.png" alt="">
+          </div>
+          <div class="description">
+            <p>utsa work experience</p>
           </div>
         </div>
       </div>
-      <div class="footer-container">
-        <div class="column1">
-          <a class="footer-item" href="#home"  onclick="animateScrollTop('about-container')">Home</a>
-          <a class="footer-item" href="#about"  onclick="animateScrollTop('about-container')">About</a>
-          <a class="footer-item" href="#work"  onclick="animateScrollTop('work-container')">My Work</a>
-          <a class="footer-item" href="#contact">Contact</a>
+      <div class="projects-container">
+        <div class="description">
+          <p>Auto Apply</p>
+          <p>This is a project built with the purpose of applying to a bunch of jobs really fast</p>
+        </div>
+        <div class="description">
+          <p>Hog Drone</p>
+          <p>This project was built with the purpose of reducing the $400 million dollars in annual damages hogs cause in Texas</p>
         </div>
-        <div class="column2">
-          <p>Contact Me!</p>
-          <input type="text" name="" placeholder="First Name">
-          <input type="text" name="" placeholder="Last Name">
-          <input type="text" name="" placeholder="Email">
-          <button type="button" name="button">Send</button>
+        <div class="description">
+          <p>Funny Driving Detection</p>
+          <p>This project was built with the purpose of catching funny people doing funny stuff while driving</p>
         </div>
       </div>
-    </body>
-  </html>
\ No newline at end of file
+    </div>
+    <div class="projects-container">
+      <div class="description"></div>
+      <div class="description"></div>
+      <div class="description"></div>
+    </div>
+  </div>
+  </div>
+  <div class="footer-container">
+    <div class="column1">
+      <a class="footer-item" href="#home" onclick="animateScrollTop('about-container')">Home</a>
+      <a class="footer-item" href="#about" onclick="animateScrollTop('about-container')">About</a>
+      <a class="footer-item" href="#work" onclick="animateScrollTop('work-container')">My Work</a>
+      <a class="footer-item" href="#contact">Contact</a>
+    </div>
+    <div class="column2">
+      <p>Contact Me!</p>
+      <p class="form-message">We'll be in touch!</p>
+      <input type="text" name="" placeholder="First Name">
+      <input type="text" name="" placeholder="Last Name">
+      <input type="text" name="" placeholder="Email">
+      <button type="button" name="button" onclick="message()">Send</button>
```

Comment:
what's this `message` function?

Reply:
added to script.js

Reply:
I'm going back on this from a previous comment, I think it's worth removing the `onclick` attribute and hooking up $(...).click(() => {}) handler via JS.
Reasons being mixing of template and logic lacks separation of concerns, it's harder to find what function actually calls message(), and it's more difficult to test.

I'm ok doing it in a separate PR though. Please open a separate PR to fix this once this PR is merged.

Reply:
will do!